pipelines/pipeline_katello_34_to_35_upgrade.yml

```diff
@@ -32,6 +32,17 @@
     - haveged
     - disable_firewall
 
+- hosts:
```

Comment:
https://github.com/theforeman/forklift/commit/9fd00b0b9c07acfa688ae243e05e1a2cc8c53497 should make this redundant but it's included. Any idea why?

Reply:
I can try without, but I was hitting an error (maybe 2 days ago before the PR was merged ;)